gradio/serializing.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import annotations
+
+import os
+from abc import ABC, abstractclassmethod
+from typing import Any, Dict
+
+from gradio import processing_utils
+
+
+class Serializable(ABC):
+    @abstractclassmethod
```

Comment:
abstractclassmethod is deprecated since 3.3. Ok to keep as is from a functional stand-point but maybe not a best practice.

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/41651716/185418246-b4f9bccb-c062-4ae7-ba87-d109f415764a.png">

Reply:
That should actually be @abstractmethod anyways, will fix

Reply:
I think we should make them static since they don't need a reference to the object inheriting these methods? In any event, thanks for making the fix!